graph/path/shortest.go

```diff
@@ -638,6 +676,10 @@ func (p AllShortest) AllBetween(uid, vid int64) (paths [][]graph.Node, weight fl
 	return paths, weight
 }
 
+// allBetween recursively constructs a slice of paths extending from the node
+// indexed into p.nodes by from to the node indexed by to. The len(seen) must
```

Comment:
I'm not sure if "The" is needed here.

Reply:
Hmmm. I read this as "The length of seen must ...". However, this isn't user-facing, so I'm not particularly fussed. I'll change it.